cmd/hydrate/src/lib.rs

```diff
@@ -0,0 +1,217 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! ## `humility hydrate`
+//!
+//! `humility hydrate` combines a raw memory dump (obtained from MGS) and a
+//! Hubris archive, forming a proper Hubris dump.
+//!
+//! ```console
+//! $ humility -a build-grapefruit-image-default.zip hydrate hubris.dry.0
```

Comment:
this is probably not a big deal, but it might be nice if we actually stated somewhere explicitly that the archive is required for this command?

Reply:
also, maybe the docs should explicitly note that the result of this command is that a new core file for the dump is written to the filesystem in the current working directory? i realize that this is the same behavior as other dump commands, but it might be nice to write it down in the docs for this command so that it's obvious what it does...

Reply:
Good idea, done